crates/common_utils/src/crypto.rs

```diff
@@ -123,6 +124,37 @@ impl VerifySignature for HmacSha256 {
     }
 }
 
+/// Represents the Base64 encoded HMAC-SHA-256 algorithm
+#[derive(Debug)]
+pub struct Base64HmacSha256;
+
+impl SignMessage for Base64HmacSha256 {
+    fn sign_message(
+        &self,
+        secret: &[u8],
+        msg: &[u8],
+    ) -> CustomResult<Vec<u8>, errors::CryptoError> {
+        let key = hmac::Key::new(hmac::HMAC_SHA256, secret);
+        let hash = hmac::sign(&key, msg);
+        let base64sign = BASE64.encode(hash.as_ref());
```

Comment:
can we not use consts::BASE64_ENGINE here? why we need new library?

Reply:
Okay, removed new library and used old one